notifications.yaml

```diff
@@ -0,0 +1,220 @@
+swagger: "2.0"
+info: 
+  version: "0.0.1"
+  title: "Notification API"
+  description: "Notification API."
+host: "localhost"
+basePath: "/api/v1"
+schemes: 
+  - "https"
+paths: 
+  /Messages/{id}: 
+    parameters: 
+      - 
+        name: "id"
+        in: "path"
+        required: true
+        type: "string"
+        format: "uuid"
+    get: 
+      operationId: "getMessage"
+      summary: "Get Message"
+      tags: 
+        - "Messages"
+      description: "Retrieve a message already submitted for delivery."
+      responses: 
```

Comment:
404 is not a common error (see my comment below) but is action-specific (only gets) and has special meaning to each endpoint (user does not exist, message does not exist, etc...)

common errors are like authentication errors (same error and meaning for all endpoints)